helix-view/src/clipboard.rs

```diff
@@ -120,6 +124,27 @@ mod provider {
         }
     }
 
+    #[cfg(target_os = "windows")]
+    #[derive(Debug)]
+    pub struct WindowsProvider;
+
+    #[cfg(target_os = "windows")]
+    impl ClipboardProvider for WindowsProvider {
+        fn name(&self) -> Cow<str> {
+            Cow::Borrowed("Windows Clipboard")
+        }
+
+        fn get_contents(&self) -> Result<String> {
+            let contents = clipboard_win::get_clipboard(clipboard_win::formats::Unicode)?;
```

Comment:
It couldn't be converted into an `anyhow::Result`

Reply:
Looks like it is though? You use `?`

Reply:
You can convert between the error types, but not the result types.